griptape/tools/inpainting_image_generation_client/tool.py

```diff
@@ -31,17 +31,11 @@ class InpaintingImageGenerationClient(BlobArtifactFileOutputMixin, BaseTool):
 
     @activity(
         config={
-            "description": "Can be used to modify an image within a specified mask area using image and mask files.",
+            "description": "Modifies an image within a specified mask area using image and mask files.",
             "schema": Schema(
                 {
-                    Literal(
-                        "prompts",
-                        description="A detailed list of features and descriptions to include in the generated image.",
-                    ): list[str],
-                    Literal(
-                        "negative_prompts",
-                        description="A detailed list of features and descriptions to avoid in the generated image.",
-                    ): list[str],
+                    Literal("prompt", description=BaseImageGenerationClient.PROMPT_DESCRIPTION): str,
+                    Literal("negative_prompt", description=BaseImageGenerationClient.NEGATIVE_PROMPT_DESCRIPTION): str,
```

Comment:
Should this be `self.PROMPT_DESCRIPTION`? Otherwise, doesn't seem like inheritance is needed.

Reply:
Inheritance is probably overkill here, but it's a place to standardize which classes image generation tools inherit from and, more importantly, provided a common space to define this repeated schema descriptions.

This class may not be necessary after addressing Collin's comment below.

Reply:
I think if they're defined as class fields, `BaseImageGenerationClient.PROMPT_DESCRIPTION` is the recommended way of accessing.

But that raises the question of whether they _should_ be class fields. I'm leaning towards adding instance fields (`prompt_description`, `negative_prompt_description`) with defaults pointing to class fields (`DEFAULT_PROMPT_DESCRIPTION`, `DEFAULT_NEGATIVE_PROMPT_DESCRIPTION`).

Reply:
I'm probably mistaken, but we don't have access to the instance in the decorator, right? I'm not sure we can pull `self.prompt_description` if it's an instance field.

Reply:
Argh you're right. You _should_ have access in the schema properties, but today you can only access it [in the schema description](https://github.com/griptape-ai/griptape/blob/39d2b95dc0bb82f43506a872cea4f0f1caf65c18/tests/mocks/mock_tool/tool.py#L17).

Ok good with how it is today.